### DISCUSSION TITLE
Audacious no longer read local .lrc file

### DISCUSSION THREAD

**Discussion by bpanca05** [audacious-media-player/audacious#93] [General]
Actually, the Audacious does read and found .lrc file on my tracks based on name matching. But on Lyrics tab, there's no lyric content / output even though Audacious match the .lrc file.

Here some logs for that: 
https://pastebin.run/cxkz-85d85vc
![image](https://github.com/audacious-media-player/audacious/assets/53902591/68f22b12-ccde-408b-9a19-c807efb3f823)

Here the expected result (Using another Audacious fork) :
![image](https://github.com/audacious-media-player/audacious/assets/53902591/abb6d7a4-bcf6-4930-a719-40c8573d9918)

**Comment by radioactiveman** [comment]
I can confirm that loading the local lrc file is not working even if the logs say it was read.

**Reply by radioactiveman** [reply]
Fixed with https://github.com/audacious-media-player/audacious-plugins/commit/9dbbf3f20f65abc7f4b0bba063b8cb172022f587.

**Comment by bpanca05** [comment]
![image](https://github.com/audacious-media-player/audacious/assets/53902591/3aa1bf1c-b50c-4f2b-9ab9-532eedc9a355)

This commit [9dbbf3f](https://github.com/audacious-media-player/audacious-plugins/commit/9dbbf3f20f65abc7f4b0bba063b8cb172022f587) fixed this issue, great job and thanks @radioactiveman
